vectormath/vector.py

```diff
@@ -189,18 +189,24 @@ class Vector2(BaseVector):
         - no input (returns [0., 0.])
     """
 
-    def __new__(cls, x=None, y=None):                                          #pylint: disable=arguments-differ
+    def __new__(cls, x=None, y=None, polar=False, unit='rad'):                 #pylint: disable=arguments-differ
```

Comment:
This call signature is fine, especially since we already lean into overloading `X`/`Y` (i.e. `X` can already be another vector or an array, so it's fine that it can also be `rho`).

However, can you add a line to the docstring for the new polar behavior? Something like:
```
- rho and theta, if polar=True; specify unit as 'rad' or 'deg'
```